int8 meta6
